examples/hello_world/lib/main.dart

```diff
@@ -4,4 +4,13 @@
 
 import 'package:flutter/widgets.dart';
 
-void main() => runApp(const Center(child: Text('Hello, world!', textDirection: TextDirection.ltr)));
+void main() =>
```

Comment:
Could you open a new PR to revert this change?

Reply:
It adds a `key` to the `Text` widget, which is needed by the test.

Reply:
Why are we reverting the change? The test are passing. I checked the smoke test after the PR went in. It always passed.